ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -19,14 +21,12 @@ StatusDialog {
     required property SwapModalAdaptor swapAdaptor
 
     objectName: "swapModal"
-    title: qsTr("Swap")
 
+    implicitWidth: 556
     bottomPadding: 16
     padding: 0
 
     background: StatusDialogBackground {
```

Comment:
You can just use `backgroundColor` on the StatusDialog to customize the color; no need to overrride the whole `background`